<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyPaint</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/учеба/iconsForPaint/color.png</normaloff>../../Desktop/учеба/iconsForPaint/color.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="m_gridLayout">
      <item row="2" column="0">
       <widget class="QGraphicsView" name="m_graphView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="m_hrzLayoutFirst">
        <item>
         <widget class="QPushButton" name="m_pbFileMenu">
          <property name="text">
           <string>File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pbMainMenu">
          <property name="text">
           <string>Main</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="m_hrzLayoutMain_2" native="true">
        <layout class="QHBoxLayout" name="m_hrzLayoutMain">
         <item>
          <widget class="QPushButton" name="m_pbPencilTool">
           <property name="text">
            <string>Pencil</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbBrushTool">
           <property name="text">
            <string>Brush</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbTextTool">
           <property name="text">
            <string>Text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbEraserTool">
           <property name="text">
            <string>Eraser</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbFigureTool">
           <property name="text">
            <string>Figure</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbColorTool">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
